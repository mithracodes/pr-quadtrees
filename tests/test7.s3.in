144.970012173013 -37.7925700784606
144.957508513963 -37.8037935434102
144.956946152453 -37.7931400061533
144.955631358526 -37.8096614186364
144.955808629929 -37.7945528716552
144.969901878645 -37.7932348609759
144.957261347093 -37.8037982238819
144.96295671685 -37.8072807821045
144.954242901089 -37.8034610850527
144.97113137655 -37.8107473912556
144.967085475717 -37.7987901401239
144.975506772005 -37.7932723402052
144.970927819817 -37.7973845653899
144.966874706153 -37.8090560013442
144.96471558029 -37.7967161908961
144.972475122992 -37.8011090220332
144.953825202782 -37.7945819686297
144.95945970235 -37.7900439279414
144.967947010621 -37.8104726708648
144.958404168274 -37.7899876779537
144.960451579308 -37.8041163018931
144.966057858604 -37.8050188486367
144.957811726048 -37.799885333796
144.955755086323 -37.8006125217302
144.970820054144 -37.80093572396
144.969689367835 -37.803022275939
144.967427964694 -37.8108837934266
144.973239439469 -37.8086926706933
144.969123915306 -37.8011224360362
144.960375749047 -37.8028415738354
144.960032645174 -37.8071681208287
144.971891602483 -37.7973078222429
144.96100367515 -37.8002349792872
144.955545354776 -37.8070525327073
144.964369880861 -37.8105994010849
144.958116779267 -37.808944492023
144.970395590224 -37.7991481414152
144.957508493994 -37.8037935002357
144.962309020339 -37.8023831843134
144.97323893652 -37.7967204520591
144.957344433483 -37.8040512018855
144.964883547517 -37.8048931001521
144.966508802792 -37.793455631383
144.972603840678 -37.8003883364151
144.960811864326 -37.8023237361582
144.979146246586 -37.8106270469818
144.96388445184 -37.8101255273973
144.956659272316 -37.7957796995232
144.958918290876 -37.8000088360761
144.966317749733 -37.8036639613274
144.968351657124 -37.8076395150506
144.961372242721 -37.791622496923
144.96173338822 -37.8095369102999
144.955271807868 -37.7966640268913
144.975432788084 -37.8108977612888
144.966366282099 -37.805049282019
144.970564244895 -37.7961558872637
144.972942894852 -37.7964129747919
144.968833537598 -37.7976319846802
144.964739110707 -37.7969936702048
144.9579542389 -37.8095933115267
144.965615915007 -37.7986311869488
144.95682556906 -37.8023885994821
144.97361327185 -37.8094325380433
144.967643407327 -37.7938286215627
144.972080523847 -37.7981702256951
144.959204470354 -37.806375062541
144.965868015087 -37.8030291894706
144.955843172564 -37.7939479033977
144.970564203366 -37.7961558773976
144.971963827592 -37.7965833399839
144.971049369829 -37.7992175291781
144.968453472502 -37.7968906274641
144.96692658187 -37.7960030261027
144.954248047657 -37.8034301260183
144.959491542437 -37.8064168170725
144.974735687797 -37.7978007597792
144.967218497866 -37.8051147943312
144.974642560902 -37.808342819118
144.955044698638 -37.8011305453007
144.969682837154 -37.8055870843759
144.974932347367 -37.7966346010201
144.957337643184 -37.7995581813658
144.9779364645 -37.8110926094238
144.962414768691 -37.8017695180845
144.965203214277 -37.8049244785178
144.967905516803 -37.8028256898531
144.957761070672 -37.8008338839896
144.965307666305 -37.8004316703235
144.957959064305 -37.805314603403
144.969334553715 -37.8065758593328
144.969060142234 -37.7995796452885
144.96900836175 -37.8101590173668
144.957198957106 -37.8103874798583
144.971733261949 -37.7992900206151
144.960521633051 -37.8020194049437
144.971872946818 -37.8082940222859
144.966303006131 -37.8033889268325
144.961202485388 -37.8065675200417
144.965640777245 -37.8025593605682
144.969904294375 -37.7932097005673
144.958032292546 -37.8038845852289
144.957645148938 -37.7926914148186
144.957443547178 -37.8095360340428
144.955724314677 -37.7950722174366
144.971048077873 -37.7933340981078
144.956595167391 -37.8036875689721
144.964217112134 -37.8069348416935
144.955388103976 -37.8035555540094
144.970808853739 -37.8101310427232
144.966777956924 -37.8005874662523
144.975315462087 -37.7943668708308
144.971517409419 -37.7974503676384
144.967167133996 -37.8096808868061
144.964895895962 -37.7961032083208
144.974144535196 -37.8012672636348
144.95506941964 -37.7936465461307
144.960968094501 -37.7915219024602
144.968771391138 -37.81025709509
144.958173757636 -37.7918544308188
144.960770644552 -37.8025539618379
144.965841566989 -37.8060614787889
144.957761070672 -37.8008338839896
144.955087603113 -37.8010956048802
144.970101511088 -37.8008602984951
144.970037409364 -37.8012247896756
144.969260739971 -37.8103733134382
144.973921654709 -37.8087588420247
144.968553481063 -37.8010569596972
144.960285084201 -37.8033571653425
144.960473779554 -37.8080001894576
144.97202467457 -37.7965898191354
144.960456462113 -37.8001757346548
144.955643859292 -37.8064934006989
144.964836641018 -37.8113922831515
144.958835806724 -37.8087717498872
144.971764044441 -37.7992709899644
144.956834594535 -37.8023364662801
144.962923894128 -37.802405625944
144.973963201346 -37.7967670982154
144.956564192013 -37.80393234923
144.964790999819 -37.8054018434854
144.965657291276 -37.7933771970883
144.973535192758 -37.8000402568815
144.961959153387 -37.8023980195576
144.978715517471 -37.8102561925536
144.963409642788 -37.8102647232653
144.955851372241 -37.7956993569423
144.958798812426 -37.8005287843
144.966133626526 -37.8043500400278
144.968621275621 -37.808048814639
144.963483989382 -37.784913661078
144.962900223832 -37.8091964380217
144.953976289699 -37.7965122776257
144.974930375205 -37.8117120869537
144.966249526027 -37.8054922655909
144.96941668057 -37.7960611657282
144.973058526381 -37.795966839001
144.968648431565 -37.798469045584
144.964486506658 -37.7982351887171
144.957437824926 -37.8095689043731
144.964478591201 -37.7985082279018
144.956909575173 -37.8018692918767
144.973119073684 -37.8094028566022
144.966805941697 -37.7937345968938
144.972632335738 -37.7982121672818
144.959686409706 -37.8070679164956
144.966400744112 -37.8028200800854
144.956542960411 -37.7931648673767
144.970641526247 -37.7957051855061
144.971836151545 -37.7972834411239
144.970395586689 -37.799148156449
144.967779058973 -37.7968300925778
144.967568352366 -37.7960737386215
144.955388109393 -37.8035555171553
144.959783364643 -37.8069828458696
144.973080369165 -37.7976541992639
144.966366291862 -37.8050492660186
144.97399934312 -37.8083094634874
144.954674538665 -37.8008352890678
144.970305648398 -37.8061996861216
144.97507597871 -37.7960180036685
144.957514339138 -37.7972103930236
144.976117408678 -37.8109210942349
144.962978989725 -37.8022512475605
144.966029511079 -37.8049817214846
144.968293192526 -37.8010278631668
144.957762100051 -37.8017122201092
144.964169809762 -37.8003077116172
144.958208876203 -37.8058043844188
144.969650877731 -37.8055913086831
144.969705769061 -37.7996469222893
144.969443572769 -37.8100303948558
144.957816979846 -37.8102327879597
144.971021823829 -37.7996856056485
144.960675889175 -37.8011448552058
144.972234132861 -37.8086832548765
144.966439732634 -37.8029385129811
144.96036488742 -37.8065059385133
144.966137001524 -37.8023152931234
