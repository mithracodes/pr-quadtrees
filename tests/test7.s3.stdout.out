144.970012173013 -37.7925700784606 --> NE SW NE NW
144.957508513963 -37.8037935434102 --> SW NW SE SW NW SE NW NE SW SE SE SW SW NW NW NW NE
144.956946152453 -37.7931400061533 --> NW SW NW SE NE NE
144.955631358526 -37.8096614186364 --> SW SW SW
144.955808629929 -37.7945528716552 --> NW SW SW NE NW NE
144.969901878645 -37.7932348609759 --> NE SW NE SW NW SW NE NW NW SE
144.957261347093 -37.8037982238819 --> SW NW SE SW NW SW NE
144.96295671685 -37.8072807821045 --> SW SE NW SE
144.954242901089 -37.8034610850527 --> SW NW SW SW NW NE NW SW SW
144.97113137655 -37.8107473912556 --> SE SW SE SW
144.967085475717 -37.7987901401239 --> SE NW NW NW
144.975506772005 -37.7932723402052 --> NE SE NW SE NE
144.970927819817 -37.7973845653899 --> NE SW SE SW
144.966874706153 -37.8090560013442 --> SE SW SW NW NW
144.96471558029 -37.7967161908961 --> NW SE SE SW NE SE NE
144.972475122992 -37.8011090220332 --> SE NW NE SE SE
144.953825202782 -37.7945819686297 --> NW SW SW NW
144.95945970235 -37.7900439279414 --> NW NW SE SE
144.967947010621 -37.8104726708648 --> SE SW SW SW NE NE
144.958404168274 -37.7899876779537 --> NW NW SE SW
144.960451579308 -37.8041163018931 --> SW NE SW SW NW SW
144.966057858604 -37.8050188486367 --> SW SE NE NE NE NW
144.957811726048 -37.799885333796 --> SW NW NE SW NE
144.955755086323 -37.8006125217302 --> SW NW NW SE
144.970820054144 -37.80093572396 --> SE NW NE SW SE
144.969689367835 -37.803022275939 --> SE NW SE
144.967427964694 -37.8108837934266 --> SE SW SW SW NE SW
144.973239439469 -37.8086926706933 --> SE SE SW NW NW
144.969123915306 -37.8011224360362 --> SE NW NW SE SE
144.960375749047 -37.8028415738354 --> SW NE SW NW SW SW
144.960032645174 -37.8071681208287 --> SW SW NE SE NE NE SE
144.971891602483 -37.7973078222429 --> NE SW SE SE SW NE NE
144.96100367515 -37.8002349792872 --> SW NE NW SW NE
144.955545354776 -37.8070525327073 --> SW SW NW SE
144.964369880861 -37.8105994010849 --> SW SE SE SW NE
144.958116779267 -37.808944492023 --> SW SW SE NW NE
144.970395590224 -37.7991481414152 --> SE NW NE NW SW NE SE NE NW SE SW SW SW SW SE SE NW SW NE NE
144.957508493994 -37.8037935002357 --> SW NW SE SW NW SE NW NE SW SE SE SW SW NW NW NW NW
144.962309020339 -37.8023831843134 --> SW NE SW NE SW NE
144.97323893652 -37.7967204520591 --> NE SE SW SW NW SW
144.957344433483 -37.8040512018855 --> SW NW SE SW NW SW SE
144.964883547517 -37.8048931001521 --> SW NE SE SW
144.966508802792 -37.793455631383 --> NE SW NW SW NW
144.972603840678 -37.8003883364151 --> SE NW NE SE NE
144.960811864326 -37.8023237361582 --> SW NE SW NW NW SE
144.979146246586 -37.8106270469818 --> SE SE SE SE NE NE
144.96388445184 -37.8101255273973 --> SW SE SE NW
144.956659272316 -37.7957796995232 --> NW SW SW NE SE
144.958918290876 -37.8000088360761 --> SW NW NE SE NW NW
144.966317749733 -37.8036639613274 --> SW NE SE SE NE NE SE
144.968351657124 -37.8076395150506 --> SE SW NW SE SW NW
144.961372242721 -37.791622496923 --> NW SE NW NW NE SE
144.96173338822 -37.8095369102999 --> SW SE SW NW
144.955271807868 -37.7966640268913 --> NW SW SW SW NE
144.975432788084 -37.8108977612888 --> SE SE SW SE NE
144.966366282099 -37.805049282019 --> SW SE NE NE NE NE NE NW NE SW SE SW NW NE SE SW
144.970564244895 -37.7961558872637 --> NE SW SE NW SE SW SW SW NE NE SE NE NW NE SE SW SW NE SE
144.972942894852 -37.7964129747919 --> NE SE SW SW NW NW
144.968833537598 -37.7976319846802 --> NE SW SW SE SW
144.964739110707 -37.7969936702048 --> NW SE SE SW NE SE SE
144.9579542389 -37.8095933115267 --> SW SW SE NW SE NW
144.965615915007 -37.7986311869488 --> SW NE NE NE
144.95682556906 -37.8023885994821 --> SW NW SW NE SE
144.97361327185 -37.8094325380433 --> SE SE SW NW SW NE
144.967643407327 -37.7938286215627 --> NE SW NW SW SE
144.972080523847 -37.7981702256951 --> SE NW NE NE NE NW
144.959204470354 -37.806375062541 --> SW SW NE NE SW
144.965868015087 -37.8030291894706 --> SW NE SE NE SE SW
144.955843172564 -37.7939479033977 --> NW SW NW SE SW
144.970564203366 -37.7961558773976 --> NE SW SE NW SE SW SW SW NE NE SE NE NW NE SE SW SW NE SW
144.971963827592 -37.7965833399839 --> NE SW SE SE NW NE SE SE NW
144.971049369829 -37.7992175291781 --> SE NW NE NW SE NE
144.968453472502 -37.7968906274641 --> NE SW SW SE NW
144.96692658187 -37.7960030261027 --> NE SW SW NW SW
144.954248047657 -37.8034301260183 --> SW NW SW SW NW NE NW SW SE
144.959491542437 -37.8064168170725 --> SW SW NE NE SE
144.974735687797 -37.7978007597792 --> NE SE SW SE SW
144.967218497866 -37.8051147943312 --> SE SW NW NW
144.974642560902 -37.808342819118 --> SE SE NW SE
144.955044698638 -37.8011305453007 --> SW NW NW SW SE SE NW NW SE
144.969682837154 -37.8055870843759 --> SE SW NE NW NW
144.974932347367 -37.7966346010201 --> NE SE SW SE NW
144.957337643184 -37.7995581813658 --> SW NW NE NW
144.9779364645 -37.8110926094238 --> SE SE SE SE NW
144.962414768691 -37.8017695180845 --> SW NE SW NE NW
144.965203214277 -37.8049244785178 --> SW NE SE SE SW
144.967905516803 -37.8028256898531 --> SE NW SW
144.957761070672 -37.8008338839896 --> SW NW NE SW SW
144.965307666305 -37.8004316703235 --> SW NE NE SE
144.957959064305 -37.805314603403 --> SW SW NE NW NE NW
144.969334553715 -37.8065758593328 --> SE SW NW NE SE
144.969060142234 -37.7995796452885 --> SE NW NW NE SE
144.96900836175 -37.8101590173668 --> SE SW SW NE SE SW
144.957198957106 -37.8103874798583 --> SW SW SE SW
144.971733261949 -37.7992900206151 --> SE NW NE NE SW NE SW SW
144.960521633051 -37.8020194049437 --> SW NE SW NW NW SW
144.971872946818 -37.8082940222859 --> SE SW NE SE
144.966303006131 -37.8033889268325 --> SW NE SE SE NE NE NE
144.961202485388 -37.8065675200417 --> SW SE NW NW SE
144.965640777245 -37.8025593605682 --> SW NE SE NE SW
144.969904294375 -37.7932097005673 --> NE SW NE SW NW SW NE NW NW NE
144.958032292546 -37.8038845852289 --> SW NW SE SW NE
144.957645148938 -37.7926914148186 --> NW SW NE NW SW
144.957443547178 -37.8095360340428 --> SW SW SE NW SW NE NW SW NE
144.955724314677 -37.7950722174366 --> NW SW SW NE NW SW
144.971048077873 -37.7933340981078 --> NE SW NE SW NE
144.956595167391 -37.8036875689721 --> SW NW SW SE NE SE
144.964217112134 -37.8069348416935 --> SW SE NE SW
144.955388103976 -37.8035555540094 --> SW NW SW SE NW NW SW NW SW SW NW NW SW NW SW SE NE NW SW
144.970808853739 -37.8101310427232 --> SE SW SE NW
144.966777956924 -37.8005874662523 --> SE NW NW SW
144.975315462087 -37.7943668708308 --> NE SE NW SE SE
144.971517409419 -37.7974503676384 --> NE SW SE SE SW NW
144.967167133996 -37.8096808868061 --> SE SW SW NW SW
144.964895895962 -37.7961032083208 --> NW SE SE NW
144.974144535196 -37.8012672636348 --> SE NE NW SW SE
144.95506941964 -37.7936465461307 --> NW SW NW SW
144.960968094501 -37.7915219024602 --> NW SE NW NW NE SW
144.968771391138 -37.81025709509 --> SE SW SW SE NW
144.958173757636 -37.7918544308188 --> NW SW NE NW NE
144.960770644552 -37.8025539618379 --> SW NE SW NW SW NE
144.965841566989 -37.8060614787889 --> SW SE NE NE SE
144.957761070672 -37.8008338839896 --> SW NW NE SW SW
144.955087603113 -37.8010956048802 --> SW NW NW SW SE SE NW NW NE
144.970101511088 -37.8008602984951 --> SE NW NE SW SW NE
144.970037409364 -37.8012247896756 --> SE NW NE SW SW SW
144.969260739971 -37.8103733134382 --> SE SW SW SE NE
144.973921654709 -37.8087588420247 --> SE SE SW NW NE
144.968553481063 -37.8010569596972 --> SE NW NW SE SW NE
144.960285084201 -37.8033571653425 --> SW NE SW SW NW NW
144.960473779554 -37.8080001894576 --> SW SE NW SW
144.97202467457 -37.7965898191354 --> NE SW SE SE NW NE SE SE NE
144.960456462113 -37.8001757346548 --> SW NE NW SW NW
144.955643859292 -37.8064934006989 --> SW SW NW NE
144.964836641018 -37.8113922831515 --> SW SE SE SW SE
144.958835806724 -37.8087717498872 --> SW SW SE NE
144.971764044441 -37.7992709899644 --> SE NW NE NE SW NE SW SE
144.956834594535 -37.8023364662801 --> SW NW SW NE NE SE
144.962923894128 -37.802405625944 --> SW NE SW NE SE
144.973963201346 -37.7967670982154 --> NE SE SW SW NE
144.956564192013 -37.80393234923 --> SW NW SW SE NE SW
144.964790999819 -37.8054018434854 --> SW SE NE NW
144.965657291276 -37.7933771970883 --> NW SE NE
144.973535192758 -37.8000402568815 --> SE NE NW SW NW
144.961959153387 -37.8023980195576 --> SW NE SW NE SW NW
144.978715517471 -37.8102561925536 --> SE SE SE SE NE NW
144.963409642788 -37.8102647232653 --> SW SE SE SW NW
144.955851372241 -37.7956993569423 --> NW SW SW NE SW
144.958798812426 -37.8005287843 --> SW NW NE SE NW SW
144.966133626526 -37.8043500400278 --> SW NE SE SE SE NE
144.968621275621 -37.808048814639 --> SE SW NW SE SW NE
144.963483989382 -37.784913661078 --> NW NE
144.962900223832 -37.8091964380217 --> SW SE SW NE
144.953976289699 -37.7965122776257 --> NW SW SW SW NW
144.974930375205 -37.8117120869537 --> SE SE SW SE SW
144.966249526027 -37.8054922655909 --> SW SE NE NE NE SE
144.96941668057 -37.7960611657282 --> NE SW SW NE
144.973058526381 -37.795966839001 --> NE SE SW NW
144.968648431565 -37.798469045584 --> SE NW NW NE NW
144.964486506658 -37.7982351887171 --> SW NE NE NW NE NW
144.957437824926 -37.8095689043731 --> SW SW SE NW SW NE NW SW SE
144.964478591201 -37.7985082279018 --> SW NE NE NW NE SW
144.956909575173 -37.8018692918767 --> SW NW SW NE NE NE
144.973119073684 -37.8094028566022 --> SE SE SW NW SW NW
144.966805941697 -37.7937345968938 --> NE SW NW SW SW
144.972632335738 -37.7982121672818 --> SE NW NE NE NE NE
144.959686409706 -37.8070679164956 --> SW SW NE SE NE NW
144.966400744112 -37.8028200800854 --> SW NE SE NE SE SE NE NW
144.956542960411 -37.7931648673767 --> NW SW NW SE NE NW
144.970641526247 -37.7957051855061 --> NE SW SE NW SE NW
144.971836151545 -37.7972834411239 --> NE SW SE SE SW NE NW
144.970395586689 -37.799148156449 --> SE NW NE NW SW NE SE NE NW SE SW SW SW SW SE SE NW SW NE SE
144.967779058973 -37.7968300925778 --> NE SW SW SW
144.967568352366 -37.7960737386215 --> NE SW SW NW SE
144.955388109393 -37.8035555171553 --> SW NW SW SE NW NW SW NW SW SW NW NW SW NW SW SE NE NW NW
144.959783364643 -37.8069828458696 --> SW SW NE SE NE NE SW
144.973080369165 -37.7976541992639 --> NE SE SW SW SW
144.966366291862 -37.8050492660186 --> SW SE NE NE NE NE NE NW NE SW SE SW NW NE SE SE
144.97399934312 -37.8083094634874 --> SE SE NW SW
144.954674538665 -37.8008352890678 --> SW NW NW SW SE NW
144.970305648398 -37.8061996861216 --> SE SW NE NW SW
144.97507597871 -37.7960180036685 --> NE SE SW NE
144.957514339138 -37.7972103930236 --> NW SW SE
144.976117408678 -37.8109210942349 --> SE SE SE SW
144.962978989725 -37.8022512475605 --> SW NE SW NE NE
144.966029511079 -37.8049817214846 --> SW NE SE SE SE SW
144.968293192526 -37.8010278631668 --> SE NW NW SE SW NW
144.957762100051 -37.8017122201092 --> SW NW SE NW
144.964169809762 -37.8003077116172 --> SW NE NE SW
144.958208876203 -37.8058043844188 --> SW SW NE NW NE SE
144.969650877731 -37.8055913086831 --> SE SW NW NE NE
144.969705769061 -37.7996469222893 --> SE NW NE NW SW SW
144.969443572769 -37.8100303948558 --> SE SW SW NE SE SE
144.957816979846 -37.8102327879597 --> SW SW SE NW SE SW
144.971021823829 -37.7996856056485 --> SE NW NE NW SE SE
144.960675889175 -37.8011448552058 --> SW NE NW SW SW
144.972234132861 -37.8086832548765 --> SE SW SE NE
144.966439732634 -37.8029385129811 --> SW NE SE NE SE SE NE SE
144.96036488742 -37.8065059385133 --> SW SE NW NW SW
144.966137001524 -37.8023152931234 --> SW NE SE NE NE
